package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-playwright",
-  "version": "1.17.110",
+  "version": "1.17.111",
```

Comment:
What's our version strategy here? Should we match playwright-core for at least the major version?

Reply:
Nope, in fact this should have been 0.0.111, but someone decided to push it as semver.